css/style.css

```diff
@@ -16,4 +15,149 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.list {
+  padding-right: 50px;
+}
+
+li {
+  display: inline;
+  padding: 15px;
+  font-size: 20px;
+  color: dimgray;
+  transition: font-size 0.5s ease;
+}
+
+li:hover {
+  color: red;
+  font-size: 25px;
```

Comment:
Nice. For links you can also change the cursor so the little hand appears:
`cursor: pointer;` 🙂